src/main/java/org/team4159/lib/Odometry.java

```diff
@@ -0,0 +1,48 @@
+package org.team4159.lib;
+
+import edu.wpi.first.wpilibj.geometry.Pose2d;
+import edu.wpi.first.wpilibj.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.util.Units;
+
+/**
+ * Tracks a physical object and its heading (i.e. a Drivetrain) on a Cartesian x, y plane, when fed encoder and gyro values.
+ */
+public class Odometry {
+  private Pose2d position;
+
+  private double prev_magnitude;
```

Comment:
is it a problem that this gets used before it's initialized?

Reply:
yeah, it should be a problem, but oddly there's never been an error